sklearn/utils/tests/test_multiclass.py

```diff
@@ -290,6 +290,13 @@ def test_type_of_target():
                ' use a binary array or sparse matrix instead.')
         assert_raises_regex(ValueError, msg, type_of_target, example)
 
+    try:
+        from pandas import SparseSeries
+        y = SparseSeries([1, 0, 0, 1, 0])
+        msg = "y cannot be class 'SparseSeries'."
+        assert_raises_regex(ValueError, msg, type_of_target, y)
+    except ImportError:
+        pass
```

Comment:
Please wrap only the import statement and use `raise SkipTest("Pandas not found")` as elsewhere